test/go.mod

```diff
@@ -0,0 +1,100 @@
+module github.com/openshift-knative/serverless-operator/test
+
+go 1.14
+
+require (
```

Comment:
Please run `go mod tidy` on this. There's way too much stuff in here.

Reply:
So, just to make sure I understand what you're asking for (and the implications).

You'd like a reduced subset of deps in go.{mod,sum} via running `go mod tidy`

However, that'll remove all the eventing bits from #521, which is fine, I'll update the title of *this* PR accordingly, but it'll make it less of a cherry pick and I'll have to rebase 521.

I understand this makes the individual PRs cleaner, just want to make sure we're on the same page and you'd still like that done here (vs in #521 where I can run the same `go mod tidy`)

either way, let me know what you'd prefer

Reply:
Just seen this comment now. Let's invert the order then. Let's merge this one as-is (if CI is happy) and then the tests.

Reply:
was asked to do this change here -- I'll rebased #521 as needed.

Reply:
@markusthoemmes just saw our comments collided -- could we please get this change in first? I've already started the work to rebase on this PR, as you suggested earlier/elsewhere, it's cleaner to make these changes here, then the eventing PR on top